man/man8/zfs.8

```diff
@@ -441,6 +441,16 @@ For non-snapshots, the compression ratio achieved for the \fBused\fR space of th
 .sp
 .ne 2
 .na
+\fB\fBcreatetxg\fR
+.ad
+.sp .6
+.RS 4n
+The transaction group (TXG) in which the dataset or bookmark was created. It is suitable for ordering a list of snapshots, e.g. for incremental \fBsend\fR & \fBrecv\fR.
```

Comment:
You can squash immediately if your changes are not so big, it will make the merge process faster.